uefi/src/proto/console/serial.rs

```diff
@@ -14,8 +14,7 @@ use bitflags::bitflags;
 /// Since UEFI drivers are implemented through polling, if you fail to regularly
 /// check for input/output, some data might be lost.
 #[repr(C)]
-#[unsafe_guid("bb25cf6f-f1d4-11d2-9a0c-0090273fc1fd")]
-#[derive(Protocol)]
```

Comment:
Can the `Protocol` derive attribute be removed from the codebase? Are there any more use cases now?